src/components/BookingModal/RejectionForm/index.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component, Fragment } from 'react';
+import { PropTypes as PT } from 'prop-types';
+import { InputText } from '../../common_styled';
+import { sendMessage } from '../../../services/holidayService';
+import messageType from '../../../utilities/messageTypes';
+import Button from '../../common/Button';
+
+export default class RejectionForm extends Component {
+  static propTypes = {
+    eventId: PT.number.isRequired,
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      rejectionMessage: '',
+    };
+  }
+
+  handleRejectionMessageChange = event => {
+    const { value } = event.target;
+    this.setState({ rejectionMessage: value });
+  };
+
+  handleSubmit = () => {
+    const { rejectionMessage } = this.state;
+    const { eventId } = this.props;
+    if (rejectionMessage !== '') {
```

Comment:
maybe better to check for a truthy value here (e.g. a non-empty string) `if (rejectionMessage) {`